Cargo.toml

```diff
@@ -10,6 +10,7 @@ members = [
   "fuel-indexer/schema",
   "xtask",
 ]
+resolver = "2"
```

Comment:
might make sense to move this to line 3, since there's an old comment there that is relevant.